test/e2e/fail_forward_e2e_test.go

```diff
@@ -102,18 +100,11 @@ var _ = Describe("Fail Forward Upgrades", func() {
 			By("updating the catalog with a broken v0.2.0 bundle image")
 			brokenProvider, err := NewFileBasedFiledBasedCatalogProvider(filepath.Join(testdataDir, failForwardTestDataBaseDir, "example-operator.v0.2.0.yaml"))
 			Expect(err).To(BeNil())
-
 			err = magicCatalog.UpdateCatalog(context.Background(), brokenProvider)
 			Expect(err).To(BeNil())
 
-			By("verifying the subscription is referencing a new installplan")
-			subscription, err = fetchSubscription(crclient, subscription.GetNamespace(), subscription.GetName(), subscriptionHasInstallPlanDifferentChecker(originalInstallPlanRef.Name))
-			Expect(err).Should(BeNil())
-
-			By("waiting for the bad InstallPlan to report a failed installation state")
-			ref := subscription.Status.InstallPlanRef
-			_, err = fetchInstallPlan(GinkgoT(), crclient, ref.Name, ref.Namespace, buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseFailed))
-			Expect(err).To(BeNil())
+			By("verifying that the subscription is still on v0.1.0")
+			Consistently(subscriptionCurrentCSVGetter(crclient, subscription.GetNamespace(), subscription.GetName())).Should(Equal("example-operator.v0.1.0"))
```

Comment:
I think this is probably not the right check for this test (and some other tests) as `subscriptionCurrentCSVGetter` checks `CurrentCSV`.